parsers/CZ.py

```diff
@@ -141,6 +141,51 @@ def __get_exchange_data(
     return exchanges.to_list()
 
 
+def get_pumping_load(
+    zone_key: ZoneKey = ZoneKey("CZ"),
+    session: Session = Session(),
+    target_datetime: datetime | None = None,
+    logger: Logger = getLogger(__name__),
+) -> list:
+    target_datetime = get_target_datetime(target_datetime)
+    from_datetime = target_datetime - timedelta(hours=48)
+
+    payload = """<?xml version="1.0" encoding="utf-8"?>
+        <soap12:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap12="http://www.w3.org/2003/05/soap-envelope">
+        <soap12:Body>
+            <Load xmlns="https://www.ceps.cz/CepsData/">
+            <dateFrom>{}</dateFrom>
+            <dateTo>{}</dateTo>
+            <agregation>{}</agregation>
+            <function>{}</function>
+            <version>{}</version>
+            </Load>
+        </soap12:Body>
+        </soap12:Envelope>""".format(
+        from_datetime.isoformat(), target_datetime.isoformat(), "QH", "AVG", "RT"
+    )
+
+    content = make_request(session, payload, zone_key).text
+    xml = BeautifulSoup(content, "xml")
+    data_tag = xml.find("data")
+    pumping_load_at_time_dict = {}
+
+    if data_tag is not None:
+        for values in data_tag:
+            pumping_load_at_time_dict[values["date"]] = float(values["value1"]) - float(
+                values["value2"]
+            )
```

Comment:
This also contains type errors. Such as `"__getitem__" method not defined on type "PageElement"`